packages/fxa-content-server/tests/functional.js

```diff
@@ -54,6 +34,17 @@ module.exports = testsSettings.concat([
   // 'tests/functional/fx_firstrun_v2.js',
   // 'tests/functional/tos.js',
   // 'tests/functional/confirm.js',
+  //'tests/functional/fx_ios_v1_sign_in.js',
+  //'tests/functional/fx_ios_v1_sign_up.js',
+  //'tests/functional/oauth_query_param_validation.js',
+  //'tests/functional/refreshes_metrics.js',
+  //'tests/functional/fx_browser_relier.js',
+  //'tests/functional/500.js',
+  //'tests/functional/email_domain_mx_validation.js',
+  //'tests/functional/post_verify/newsletters.js',
+  //''tests/functional/robots_txt.js',
```

Comment:
nit, there's two single quotes here here. It doesn't really matters since it's commented out though... same thing on line 47.